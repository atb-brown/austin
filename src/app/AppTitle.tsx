import Link from "../component/Link";
import logo from "../logo.svg";
import React from "react";

/**
 * The title block of the app, complete with logo.
 *
 * @return {ReactElement}
 */
export default function AppTitle() {
  const title = "Austin's React Website.";

  return (
    <div style={{ display: "flex", alignItems: "flex-start", width: "100%" }}>
      <img alt="logo" className="App-logo" src={logo} />
      <div>
        {title}
        <br />
        <Link
          description="(see the source code)"
          url="https://github.com/atb-brown/austin"
        />
      </div>
    </div>
  );
}
