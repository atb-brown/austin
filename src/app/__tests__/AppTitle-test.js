import renderer from "react-test-renderer";
import AppTitle from "../AppTitle";

it("renders to match snapshot", () => {
  const tree = renderer.create(<AppTitle />);

  expect(tree).toMatchSnapshot();
});
