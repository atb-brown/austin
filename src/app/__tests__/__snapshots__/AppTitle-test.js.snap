// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders to match snapshot 1`] = `
<div
  style={
    {
      "display": "flex",
      "width": "100%",
    }
  }
>
  <img
    alt="logo"
    className="App-logo disable-text-selection"
    src={{}}
  />
  <div>
    Austin's React Website
    <br />
    <a
      className="App-link"
      href="https://github.com/atb-brown/austin"
    >
      (see the source code)
    </a>
  </div>
</div>
`;
